test-utils/src/main/java/apoc/trigger/TriggerTestUtil.java

```diff
@@ -0,0 +1,26 @@
+package apoc.trigger;
+
+import org.neo4j.graphdb.GraphDatabaseService;
+
+import java.util.Map;
+
+import static apoc.util.TestUtil.testCallEventually;
+import static org.junit.Assert.assertEquals;
+
+public class TriggerTestUtil {
+    public static final long TIMEOUT = 10L;
+    public static final long TRIGGER_DEFAULT_REFRESH = 3000;
```

Comment:
Could this be a bit lower perhaps? I don't see a reason to wait 3 seconds each time and it will only make the tests slower - I was thinking that perhaps 500 or 250 would be more appropriate.

Reply:
I changed it [here](https://github.com/neo4j-contrib/neo4j-apoc-procedures/pull/3355/files#diff-346dafcb30d4bb1af05187a8d6c810d6bbf9779c222dd1c3c36bcfce44879d4aR74).

But, as mentioned there, I'm not sure why this thing seems to de-flake the test.

I could try to reduce the refresh here, maybe in 5.x it's not flaky.

Reply:
Thanks for making the change. If we notice that it's flaky then we could potentially either investigate why a lower refresh rate is problematic, or we could bump this value back up!